core/src/broadcast_stage.rs

```diff
@@ -44,6 +44,9 @@ pub(crate) mod broadcast_utils;
 mod fail_entry_verification_broadcast_run;
 mod standard_broadcast_run;
 
+const CLUSTER_NODES_CACHE_CAP: usize = 8; // Number of epochs.
```

Comment:
s/CLUSTER_NODES_CACHE_CAP/CLUSTER_NODES_CACHE_NUM_EPOCH_CAP/ and remove the comment?

Reply:
done